pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -575,6 +604,40 @@ func addResourceTargetInfo(resource pcommon.Resource, settings Settings, timesta
 	addSample(tsMap, sample, labels, infoType)
 }
 
+func addScopeInfo(
+	resource pcommon.Resource,
+	scope pcommon.InstrumentationScope,
+	settings Settings,
+	timestamp pcommon.Timestamp,
+	tsMap map[string]*prompb.TimeSeries,
+) {
+	if settings.DisableScopeInfo {
+		return
+	}
+
+	// Only add service name and instance id resource attributes
+	resCopy := pcommon.NewResource()
+	serviceName, ok := resource.Attributes().Get(conventions.AttributeServiceName)
+	if ok {
+		serviceName.CopyTo(resCopy.Attributes().PutEmpty(conventions.AttributeServiceName))
+	}
+	serviceInstanceID, ok := resource.Attributes().Get(conventions.AttributeServiceInstanceID)
+	if ok {
+		serviceInstanceID.CopyTo(resCopy.Attributes().PutEmpty(conventions.AttributeServiceInstanceID))
+	}
+
+	name := scopeMetricName
+	if len(settings.Namespace) > 0 {
+		name = settings.Namespace + "_" + name
+	}
+	labels := createAttributes(resource, scope, scope.Attributes(), settings.ExternalLabels, nameStr, name)
```

Comment:
```suggestion
	labels := createAttributes(resCopy, scope, scope.Attributes(), settings.ExternalLabels, nameStr, name)
```

Reply:
However I think removing the namespace from the resource attributes is incorrect. 

According [to the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.25.0/specification/compatibility/prometheus_and_openmetrics.md#resource-attributes-1) namespace should be combined with the service name to form the job label.

>In OTLP, the service.name, service.namespace, and service.instance.id triplet is [required to be unique](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.25.0/specification/resource/semantic_conventions/README.md#service), which makes them good candidates to use to construct job and instance. In the collector Prometheus exporters, the service.name and service.namespace attributes MUST be combined as <service.namespace>/<service.name>, or <service.name> if namespace is empty,

`createAttributes` is already only considering the service.name, service.namespace, and service.instance.id triplet for resource attributes, therefore there is no need to create `resCopy`?

Reply:
It would seem you are correct that `resCopy` is not necessary. I have removed its declaration.